packages/pyright-internal/src/tests/harness/fourslash/testLanguageService.ts

```diff
@@ -88,6 +88,8 @@ export class TestLanguageService implements LanguageServerInterface {
     constructor(workspace: WorkspaceServiceInstance, readonly console: ConsoleInterface, readonly fs: FileSystem) {
         this._workspace = workspace;
         this._uriParser = new UriParser(this.fs);
+        const defaultTestConfig = new ConfigOptions('.');
+        defaultTestConfig.compactSignatureDisplay = true;
```

Comment:
todo: will remove. logic is in `_applyTestConfigOptions` to set `compactSignatureDisplay ` to `True`